### PR TITLE
Add NVIDIA GPU operator as bundled App

### LINKED ISSUE #13017: Add Nvida Operator as default Application
https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/getting-started.html

Install the Operator with the default configuration:

```helm install --wait --generate-name  -n gpu-operator --create-namespace nvidia/gpu-operator```

### PR DESCRIPTION
**Pull request by SimonTheLeg**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #13017 

**What type of PR is this?**

/kind feature

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change? Then add your Release Note here**:

```release-note
Add nvidia-gpu-operator to default apps
```

**Documentation**:

```documentation
https://github.com/kubermatic/docs/pull/1643
```

### CODE REVIEW

File: pkg/ee/default-application-catalog/applicationdefinitions/nvidia-gpu-operator-app.yaml [hunk 1]
```diff
@@ -0,0 +1,236 @@
+#                Kubermatic Enterprise Read-Only License
+#                       Version 1.0 ("KERO-1.0”)
+#                   Copyright © 2023 Kubermatic GmbH
```
Comment (ahmedwaleedmalik) [inline pkg/ee/default-application-catalog/applicationdefinitions/nvidia-gpu-operator-app.yaml]: `s/2023/2024`

**Review by ahmedwaleedmalik — CHANGES_REQUESTED** [review]

File: pkg/ee/default-application-catalog/applicationdefinitions/nvidia-gpu-operator-app.yaml (hunk 1 above)
Comment (SimonTheLeg) [reply pkg/ee/default-application-catalog/applicationdefinitions/nvidia-gpu-operator-app.yaml]: fixed

**Review by ahmedwaleedmalik — APPROVED** [review]
/approve

**Pull request merged**